cmd/gitops/ui/run/cmd.go

```diff
@@ -54,21 +61,33 @@ func runCmd(cmd *cobra.Command, args []string) error {
 	assetHandler := http.FileServer(http.FS(assetFS))
 	redirector := createRedirector(assetFS, log)
 
-	cfg, err := server.DefaultConfig()
+	appConfig, err := server.DefaultApplicationsConfig()
 	if err != nil {
 		return fmt.Errorf("could not create http client: %w", err)
 	}
 
 	if !loggingEnabled {
-		cfg.Logger = zapr.NewLogger(zap.NewNop())
+		appConfig.Logger = zapr.NewLogger(zap.NewNop())
+	}
+
+	rest, clusterName, err := kube.RestConfig()
+	if err != nil {
+		return fmt.Errorf("could not create client config: %w", err)
+	}
+
+	_, rawClient, err := kube.NewKubeHTTPClientWithConfig(rest, clusterName)
+	if err != nil {
+		return fmt.Errorf("could not create kube http client: %w", err)
 	}
 
-	appsHandler, err := server.NewApplicationsHandler(context.Background(), cfg)
+	profilesConfig := server.NewProfilesConfig(rawClient, helmRepoNamespace, helmRepoName)
+
+	appAndProfilesHandlers, err := server.NewApplicationsAndProfilesHandler(context.Background(), &server.Config{AppConfig: appConfig, ProfilesConfig: profilesConfig})
```

Comment:
Yeah, I know what this is doing, but it's a weird thing bringing these together.

Usually an `And` in a function doesn't mean a good thing. :D It means it isn't doing just one thing. :)

Reply:
I mean it could just be `NewHandlers` I guess?

Reply:
That makes sense, yes.

Reply:
I also prefer not having an `And` in the func. `NewHandler` works. :)

Reply:
I changed it to `NewHandlers` since it's a bit better than having `And` in the function